lib/solidus_paypal_commerce_platform/client.rb

```diff
@@ -41,6 +44,21 @@ def i18n_scope_for(request)
       "solidus_paypal_commerce_platform.responses.#{request.class.name.underscore}"
     end
 
+    def self.fetch_api_credentials(auth_code:, client_id:, nonce:)
+      client = new(client_id: client_id)
+
+      access_token = client.execute(AccessTokenAuthorizationRequest.new(
+        environment: client.environment,
+        auth_code: auth_code,
+        nonce: nonce,
+      )).result.access_token
```

Comment:
This call is returning unauthorized for me locally, and causing the below exception, while trying to create a new payment method through the wizard. Does that same happen for you?

![image](https://user-images.githubusercontent.com/5720486/89547079-eeeb8700-d7ca-11ea-8805-1bf500d598b9.png)

Reply:
Fixed, the nonce I was generating was too short 😬